demisto_sdk/main.py

```diff
@@ -1,11 +1,241 @@
+    "-o", "--outdir", help="The output dir to write the unified yml to", required=True
+)
+def unify(**kwargs):
+    unifier = Unifier(**kwargs)
+    return unifier.merge_script_package_to_yml()
+
+
+@main.command(name="validate",
+              short_help='Validate your content files.')
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    '-j', '--conf-json', is_flag=True,
+    default=False, help='Validate the conf.json file.')
+@click.option(
+    '-i', '--id-set', is_flag=True,
+    default=False, help='Create the id_set.json file.')
+@click.option(
+    '-p', '--prev-ver', help='Previous branch or SHA1 commit to run checks against.')
+@click.option(
+    '-c', '--circle', type=click.Choice(["True", "False"]), default='False',
+    help='Is CircleCi or not')
+@click.option(
+    '-b', '--backward-comp', type=click.Choice(["True", "False"]), default='True',
+    help='To check backward compatibility.')
+@click.option(
+    '-g', '--use-git', is_flag=True,
+    default=False, help='Validate changes using git - this will check your branch changes and will run only on them.')
+@pass_config
+def validate(config, **kwargs):
+    sys.path.append(config.configuration.env_dir)
+
+    validator = FilesValidator(configuration=config.configuration, is_backward_check=str2bool(kwargs['backward_comp']),
+                               is_circle=str2bool(kwargs['circle']), prev_ver=kwargs['prev_ver'],
+                               validate_conf_json=kwargs['conf_json'], use_git=kwargs['use_git'])
+    return validator.run()
+
+
+@main.command(name="create",
+              short_help='Create content artifacts.')
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    '-a', '--artifacts_path', help='The path of the directory in which you want to save the created content artifacts')
+@click.option(
+    '-p', '--preserve_bundles', is_flag=True, default=False,
+    help='Flag for if you\'d like to keep the bundles created in the process of making the content artifacts')
+def create(**kwargs):
+    content_creator = ContentCreator(**kwargs)
+    content_creator.run()
+
+
+@main.command(name="secrets",
+              short_help="Run Secrets validator to catch sensitive data before exposing your code to public repository."
+                         " Attach full path to whitelist to allow manual whitelists. Default file path to secrets is "
+                         "'./Tests/secrets_white_list.json' ")
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    '-c', '--circle', type=click.Choice(["True", "False"]), default='False',
+    help='Is CircleCi or not')
+@click.option(
+    '-wl', '--whitelist', default='./Tests/secrets_white_list.json',
+    help='Full path to whitelist file, file name should be "secrets_white_list.json"')
+@pass_config
+def secrets(config, **kwargs):
+    sys.path.append(config.configuration.env_dir)
+    validator = SecretsValidator(configuration=config.configuration, is_circle=str2bool(kwargs['circle']),
```

Comment:
why `secret_white_list.json` has a configuration but `conf.json` and `id_set.json` in validate doesn't?

Reply:
should be an enhancement to those commands